content/en/blog/grpc-csharp-future.md

```diff
@@ -1,193 +1,82 @@
-a just-released .NET Core framework, while the original gRPC C# implementation
-had been stable for a long time, had lots of users and worked on even very old
-.NET Framework versions. Things needed some time to settle.
-
-Since then, the new grpc-dotnet implementation has come a long way: it has been
-adopted by many users and became very popular, it has been used by a lot of
-applications in production environments, and has also added a lot of interesting
-new features. In addition, its main prerequisite, the .NET Core 3 framework, has
-been around for a while now and its adoption numbers are growing.
-
-At the same time, while the [original gRPC C#
-implementation](https://github.com/grpc/grpc/tree/master/src/csharp) (often
-referred to as "Grpc.Core", the name of its nuget package) definitely has its
-place and it is hugely popular, we are now nearing a point where some of the
-design decisions that made perfect sense back in 2016 (when gRPC C# was released
-as GA) no longer have the weight they used to. For example, we decided to base
-the gRPC C# implementation on a native library because in 2016, there was no
-usable C# HTTP/2 library that we could depend on. By depending on the C core
-native library instead, we were able to deliver a stable, high performance gRPC
-library much faster than if we had to implement everything in C# from scratch.
-But from today's perspective, taking a native dependency doesn't make that much
-sense anymore since HTTP/2 support is now built into the .NET Core framework.
-The benefits of having a native dependency are diminishing, while the
-maintenance burden of having one is staying the same.
-
-Out of the two stable C# implementations, the grpc-dotnet implementation is
-definitely the one that has more future potential. It is a more modern
-implementation that is well-integrated with the modern versions of .NET and it's
-likely going to be more aligned with where the C# community will be a few years
-from now. It is also a pure C# implementation (no native components), which
-makes it much more contribution friendly, leads to better debuggability and it
-is simply also something that C# enthusiasts like to see.
-
-Because the maintenance costs of having two official implementations of gRPC for
-C# are nontrivial and because grpc-dotnet seems to be the best choice for all
-users in the long run, we would like to **announce the intent to phase out the
-original gRPC C# implementation (nuget package Grpc.Core) in favor of the more
-modern and more forward-looking grpc-dotnet implementation**.
-
-The details of the plan are described in the following sections, along with
-further explanation why it makes sense. To help understand the consequences of
-the decision to phase out Grpc.Core, we have also come up with a list of
-frequently asked questions and provided answers to them.
-
-### What makes grpc-dotnet the preferred implementation
```

Comment:
I'd like to keep the sections "What makes grpc-dotnet the preferred implementation" and "Why not keep Grpc.Core forever" in some form  (either keep it as a section in the blogpost, include it in the q/a, or make it an appendix). I feel like the information provided in these sections is important and while you did a good job summarizing it, we should still make the propositions explain in details somewhere in the text.